koala/cogs/base/api.py

```
@@ -89,6 +106,113 @@ async def post_schedule_activity(self, activity_type, message, url, start_time,
 
+    @parse_request
+    async def get_ping(self):
+        """
+        Get the latency of the bot
+        :return: The ping
+        """
+        return await core.ping(self._bot)
+
+    @parse_request()
+    async def get_support_link(self):
+        """
+        Get the support link of KoalaBot
+        :return: The support link
+        """
+        return core.support_link()
+
+    @parse_request
+    async def get_version(self):
+        """
+        Get the version of KoalaBot
+        :return: The version
+        """
+        return core.get_version()
+
+    @parse_request(raw_response=True)
+    async def post_load_cog(self, extension, package):
+        """
+        Loads a cog from the cogs folder
+        :param extension: name of the cog
+        :param package: package of the cogs
+        :return:
+        """
+        try:
+            await core.load_cog(self._bot, extension, package)
+        except BaseException as e:
+            error = 'Error loading cog: {}'.format(handleActivityError(e))
+            logger.error(error)
+            raise web.HTTPUnprocessableEntity(reason="{}".format(error))
+        
+        return build_response(OK, {'message': 'Cog loaded'})
+
+    @parse_request(raw_response=True)
+    async def post_unload_cog(self, extension, package):
+        """
+        Unloads a cog from the cogs folder
+        :param extension: name of the cog
+        :param package: package of the cogs
+        :return:
+        """
+        try:
+            resp = await core.unload_cog(self._bot, extension, package)
+        except BaseException as e:
+            error = 'Error unloading cog: {}'.format(handleActivityError(e))
+            logger.error(error)
+            raise web.HTTPUnprocessableEntity(reason="{}".format(error))
+        
+        if resp == "Sorry, you can't unload the base cog":
+            return build_response(BAD_REQUEST, {'message': "Sorry, you can't unload the base cog"})
```

This should throw an error instead of creating a response